src/components/subcomponents/join-team-button.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Button } from "@material-ui/core";
+
+export function JoinButton(props) {
+  const onClick = event => {
+    event.preventDefault();
+    props.joinHack();
+  };
+  if (props.team !== undefined && props.team.includes(props.userId)) {
+    return (
+      <Button variant="contained" color="primary" disabled>
+        Join
+      </Button>
+    );
+  } else {
+    return (
+      <Button
+        variant="contained"
+        color="primary"
+        onClick={onClick}
+        disabled={props.team && props.team.includes(props.userId)}
```

Comment:
do you still need the if/else if you're setting the `disabled` prop with the same condition?